scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-gating-template.yaml

```diff
@@ -1,5 +1,5 @@
 - job-template:
-    name: 'cloud-mkcloud{version}-gating'
+    name: 'cloud-mkcloud{version}-{arch}-gating'
```

Comment:
We are not. This is just a rename - to have a consistent naming scheme. And it makes it explicit what we are testing.

And yes, it (the rename) allows to have gating run on other architectures in the future as well.